### ISSUE TITLE
Build errors when configured for Python wrapping

### ISSUE THREAD

**Issue by mseng10** [InsightSoftwareConsortium/ITK#1214]
### Description
ITK build incomplete due to errors when wrapping with Python.

### Steps to Reproduce

Configure ITK with the following:

- ITK_LEGACY_REMOVE = OFF

- ITK_FUTURE_LEGACY_REMOVE = OFF

- ITK_LEGACY_SILENT = ON

- ITK_WRAP_PYTHON = ON

### Expected behavior
Clean build.

### Actual behavior
`itkAdditiveGaussianNoiseMeshFilter: warning(4): ITK type not wrapped, or currently not known: itk::MeshToMeshFilter< itk::Mesh< float, 2, itk::DefaultDynamicMeshTraits< float, 2, 2, float, float, float > >, itk::Mesh< float, 2, itk::DefaultDynamicMeshTraits< float, 2, 2, float, float, float `

`/Users/mseng/ITK-build-no-legacy/Wrapping/Typedefs/itkAdditiveGaussianNoiseMeshFilter.i:124: Warning 401: Nothing known about base class 'itk::MeshToMeshFilter< itk::Mesh< float,3,itk::DefaultDynamicMeshTraits< float,3,3,float,float,float > >,itk::Mesh< float,3,itk::DefaultDynamicMeshTraits< float,3,3,float,float,float > > >'. Ignored.`

`/Users/mseng/ITK-build-no-legacy/Wrapping/Typedefs/itkAdditiveGaussianNoiseMeshFilter.i:124: Warning 401: Maybe you forgot to instantiate 'itk::MeshToMeshFilter< itk::Mesh< float,3,itk::DefaultDynamicMeshTraits< float,3,3,float,float,float > >,itk::Mesh< float,3,itk::DefaultDynamicMeshTraits< float,3,3,float,float,float > > >' using %template.`

`make[2]: *** [Wrapping/Modules/MeshNoise/itkAdditiveGaussianNoiseMeshFilterPython.cpp] Error 4
make[2]: *** Deleting file Wrapping/Modules/MeshNoise/itkAdditiveGaussianNoiseMeshFilterPython.cpp'
make[1]: *** [Wrapping/Modules/MeshNoise/CMakeFiles/MeshNoisePython.dir/all] Error 2`

### Versions
Latest release of ITK python package. 
Latest release of ITK (master branch) on Github.

### Environment
Done on a Mac. Python 3.7 used in virtual environment.

**Comment by thewtex** [comment]
@mseng10 I tried to reproduce this with the described CMake options, ITKPythonPackage, and ITK Git master on a Mac with Python 3.7. Did you try a clean build? Any other changes in your build?

**Comment by mseng10** [comment]
@thewtex I have isolated the problem. In a separate build directory I enabled the module: `Module_MeshNoise` which subsequently downloaded the remote repository to my local source ITK. On my current build, this configuration was turned `OFF`, but MeshNoise was still part of my local repository. When attempting to build, I got the errors stated earlier. 

I attempted a clean build, and created a new build directory, but neither worked. My understanding is the lone presence of the MeshNoise directory caused CMake to look for something that was not configured to be there, hence the error. Since removing the MeshNoise directory I have gotten a clean build. Thank you for checking on the issue, much appreciated!
